desktop/apps/artwork/components/partner_stub/index.jade

```diff
@@ -9,6 +9,8 @@ unless artwork.is_in_auction
         = artwork.collecting_institution
 
     else
+      if !artwork.is_for_sale && !artwork.is_inquireable && !artwork.is_acquireable
```

Comment:
It seems like artworks that are on hold are not for sale but still shows prices and the contact form and `!artwork.is_inquireable` worked well in this case.